supplemental-ui/partials/header-content.hbs

```diff
@@ -42,9 +42,7 @@
             <a class="navbar-item" href="https://www.opennms.com/sponsored-development" target="blank">Sponsored Development</a>
           </div>
         </div>
-        <div class="navbar-item">
-          <span class="control">
-            <a class="button is-primary" href="https://github.com/opennms" target="blank">GitHub</a>
```

Comment:
I would keep the link to our GitHub opennms repository. The edit this page is really specifically editing the page. The GitHub link goes to the source code in general and allows to people to browse on their own or pick release branches or whatever.

Reply:
@indigo423 , that makes a lot of sense. I've added the button back in.